sql/migrations/20241212154910_v0.53.0.sql

```diff
@@ -0,0 +1,61 @@
+-- Create enum type "MessageQueueItemStatus"
+CREATE TYPE "MessageQueueItemStatus" AS ENUM('PENDING', 'ASSIGNED');
+
+-- Create "MessageQueue" table
+CREATE TABLE
+    "MessageQueue" (
+        "name" text NOT NULL,
+        "lastActive" TIMESTAMP(3) DEFAULT CURRENT_TIMESTAMP,
+        "durable" boolean NOT NULL DEFAULT true,
+        "autoDeleted" boolean NOT NULL DEFAULT false,
+        "exclusive" boolean NOT NULL DEFAULT false,
+        "exclusiveConsumerId" uuid NULL,
+        PRIMARY KEY ("name")
+    );
+
+-- Create "MessageQueueItem" table
+CREATE TABLE
+    "MessageQueueItem" (
+        "id" bigint NOT NULL GENERATED ALWAYS AS IDENTITY,
```

Comment:
assuming we're always using id for order?
i think we needed to increase timestamp resolution if we are using it for order for other events

Reply:
Ordering isn't strictly guaranteed, so I'm not sure why this would be necessary?